microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -115,6 +115,24 @@ $ ip route
 ----
 This routing rule matches the Kubernetes service IP address range and forwards the packet to the gateway bridge `br-ex`. You must enable `ip_forward` on the host. After the packet is forwarded to the OVS bridge `br-ex`, it is handled by OpenFlow rules in OVS. OpenFlow then steers the packet to the OVN network and eventually to the pod.
 
+[id="microshift-4-14-notable-technical-changes"]
+== Notable technical changes
+
+{product-title} {product-version} introduces the following notable technical changes.
+// Note: use [discrete] for these sub-headings.
+
+[discrete]
+[id="microshift-4-14-networking-changes"]
+=== Networking changes
+
+Networking changes to {product-title} {product-version} include IP address assignment changes, gateway changes, and other refinements.
```

Comment:
"..., and other refinements" reads a bit vague. Consider providing specific information.